dts/bindings/sensor/st,lis2dw12-common.yaml

```diff
@@ -48,6 +48,28 @@ properties:
         -  4
         -  2
 
+    odr:
+      type: int
+      required: false
+      description: |
+        Output data rate in Hz. Default value is 12.5Hz if this is not set.
```

Comment:
Why was this default value chosen? Is it a power on reset register value?

Reply:
The output data rate was 12.5 Hz before the my commits. So i gave this value with DT_INST_PROP OR macro. 12.5hz chosen because it is the good value with power consumption and performance.